src/view/components/rule-engine/rule-component.ts

```diff
@@ -38,7 +38,7 @@ var rsrc = {
   selector: 'rule'
 })
 @View({
-  template: `<div flex layout="column" class="cw-rule" [class.cw-hidden]="hidden" [class.cw-disabled]="!rule.enabled">
+  template: `<div class="cw-rule" [class.cw-hidden]="hidden" [class.cw-disabled]="!rule.enabled">
   <div flex layout="row" class="cw-header" *ngIf="!hidden" (click)="toggleCollapsed()">
```

Comment:
We might be able to remove the 'flex' from here (but leave the 'layout="row"'). Maybe. If you're still playing it's worth a try.